examples/messaging/jms-websocket-mp/src/main/java/io/helidon/messaging/mp/example/Main.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c)  2020 Oracle and/or its affiliates.
```

Comment:
There is no need to use a `Main` class in MP. If you remove it (and the property from `pom.xml`) it will work fine.
The default main class is `io.helidon.microprofile.cdi.Main` - but you do not need to explicitly define it